serverless.js

```diff
@@ -20,6 +20,9 @@ class CloudFront extends Component {
     this.context.status('Deploying')
 
     inputs.region = inputs.region || 'us-east-1'
+    inputs.enabled = inputs.enabled === false ? false : true
+    inputs.comment =
+      inputs.comment === null || inputs.comment === undefined ? '' : String(inputs.comment)
```

Comment:
```suggestion
      typeof inputs.comment === 'string' ? inputs.comment : ''
```

Reply:
My intention was to leave possibility to put numbers or arrays in comments. It is really easy to forget that your need to quote numbers and in such situation your comment just will not appear

Reply:
That’s fair enough 🙂